docs/source/user_guide/pipelines.ipynb

```diff
@@ -32,7 +32,10 @@
     "from evalml.pipelines import MulticlassClassificationPipeline\n",
     "\n",
     "class CustomMulticlassClassificationPipeline(MulticlassClassificationPipeline):\n",
-    "    component_graph = ['Imputer', 'Random Forest Classifier']"
+    "    component_graph = ['Imputer', 'Random Forest Classifier']\n",
```

Comment:
Minimal code to get this notebook to pass. However, it's worth discussing if any of this is even necessary anymore--we spend a majority of this notebook talking about different attributes that can be overridden in the pipeline class, but this can be done in the constructor in one fell swoop now. Still, I would prefer to file a separate issue for this if we decide to merge this spike.